core/src/main/java/com/gentics/mesh/core/endpoint/schema/SchemaEndpoint.java

```diff
@@ -202,6 +203,25 @@ private void addReadHandlers() {
 			InternalActionContext ac = wrap(rc);
 			crudHandler.handleReadList(ac);
 		}, false);
+	}
 
+	private void addMiscHandlers() {
+		InternalEndpointRoute readOne = createRoute();
+		readOne.path("/:schemaUuid/projects");
+		readOne.addUriParameter("schemaUuid", "Uuid of the schema.", SCHEMA_VEHICLE_UUID);
+		readOne.method(GET);
+		readOne.description("Load the projects, attached to the schema with the given uuid.");
+		readOne.exampleResponse(OK, projectExamples.getProjectListResponse(), "Loaded projects.");
+		readOne.addQueryParameters(GenericParametersImpl.class);
```

Comment:
Endpoint also supports the PagingParameters (at least the paging part), so we should add them here also